src/index.html

```diff
@@ -2,15 +2,62 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
+      method="post"
+      data-qa="big"
+    >
+      <div class="searchContainer--big">
+        <button type="submit" class="searchButton--big">
+          <img
+            class="searchIcon--big"
+            src="images/Search.svg"
+            alt="magnifying glass"
+          >
+          </img>
+        </button>
+        <div class="inputWrapper">
```

Comment:
do you need this wrapper?

Reply:
This wrapper expands our input width to 100% automaticly.

Reply:
I agree with comment above. It's redundant wrapper

Reply:
Wrapper removed